code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm

```diff
@@ -346,6 +346,22 @@
 
 //  F CKEYS
 
+//Foxicide
+//Kaith
+/datum/gear/fluff/kaith_knife
+	path = /obj/item/weapon/kitchenknife/tacknife/unathiknife/fluff/kaith
```

Comment:
needs to be a modkit, from plasteel survival knife to that path. Should have some modkit examples in custom items and such files.